src/helpers/promise.js

```diff
@@ -1,4 +1,5 @@
 /* eslint no-proto:0 */
+/* eslint no-use-before-define:0 */
```

Comment:
Ah, this is a different issue than I thought at first (sorry I was lazy and didn't look at the code).

You just need to move https://github.com/sane/sane/pull/208/files#diff-7f3b0889342e268bb3d7b66cab6926abR8 to the end of the file, because `PromiseExt` is not a global variable, it's just defined down on https://github.com/sane/sane/pull/208/files#diff-7f3b0889342e268bb3d7b66cab6926abR33.